### PR TITLE
Rescue from Net::OpenTimeout failures

### PR DESCRIPTION
**Pull request by tduffield**

### CODE REVIEW

File: lib/license_scout/license.rb [hunk 1]
```diff
@@ -107,7 +107,7 @@ def license_content(license_id, contents_url)
         begin
           LicenseScout::Log.debug("[license] Pulling license content for #{license_id} from #{new_url}")
           open(new_url).read
-        rescue OpenURI::HTTPError, Errno::ECONNREFUSED, SocketError
+        rescue OpenURI::HTTPError, Errno::ECONNREFUSED, SocketError, Net::OpenTimeout
```
Comment (phiggins) [inline lib/license_scout/license.rb]: Is there any reason to not do this for all errors?

Comment (tduffield) [reply lib/license_scout/license.rb]: Eh, probably not. I trend towards being specific but I think it's probably safe.

File: lib/license_scout/license.rb [hunk 2]
```diff
@@ -107,7 +107,7 @@ def license_content(license_id, contents_url)
         begin
           LicenseScout::Log.debug("[license] Pulling license content for #{license_id} from #{new_url}")
           open(new_url).read
-        rescue OpenURI::HTTPError, Errno::ECONNREFUSED, SocketError
+        rescue
```
Comment (phiggins) [inline lib/license_scout/license.rb]: This is going to break the redirection logic in the rescue block below.

Comment (tduffield) [reply lib/license_scout/license.rb]: gah, you're right

**Pull request merged**